torch/testing/_internal/common_methods_invocations.py

```diff
@@ -16576,7 +16576,7 @@ def sample_inputs_alias_copy(op_info, device, dtype, requires_grad, **kwargs):
     ),
     BinaryUfuncInfo('nextafter',
                     dtypes=floating_types_and(torch.bfloat16, torch.half),
-                    supports_autograd=False,
+                    supports_autograd=True,
```

Comment:
```suggestion
                    supports_forward_ad=True,
                    supports_fwgrad_bwgrad=True,
```
supports_autograd is already by default True, so we don't need it anymore
we should also enable testing for forward AD though since we've added the jvp
I also expect  forward over backward to be supported automatically